frontend/src/lib/utils.tsx

```diff
@@ -714,6 +714,22 @@ export function eventToDescription(
     return event.event
 }
 
+const eventTypesMap: { [key: string]: string } = {
```

Comment:
nit that `event type` has at least one meaning already so i'd be tempted to call this `eventToVerb` or something similar